index.js

```diff
@@ -132,8 +147,9 @@ app.post('/v1/autodock', (req, res) => {
             res.send(response);
         }
         catch(error) {
-            res.status(400)
             errorMsg = 'Execution error: ' + error;
+            res.status(500)
+            return res.send(errorMsg)
```

Comment:
Same as above

Reply:
in the `form.on('end'`, it's not going to be duplicated. It'll just return and end.